test/parallel/test-abortcontroller.js

```diff
@@ -231,6 +233,30 @@ const { setTimeout: sleep } = require('timers/promises');
   AbortSignal.timeout(1_200_000);
 }
 
+{
+  (async () => {
```

Comment:
I am not sure why this requires the async iife wrapper, or the sleeps

Reply:
without the sleep the test would fail (as I assume the GC happen in a different thread)

Reply:
@rluvaton so is 10ms trust-worthy?

Reply:
I used 10ms like the rest

Reply:
increased to 100ms and hope the test now pass

Reply:
I replaced the test with another one that check that the memory leak warning not emitted which would be faster and have the same result